src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,63 @@
+.nav {
+  background-color: $color-grey;
+  padding: $banner__head-height 20px 20px;
+
+  position: absolute;
+  top: 0;
+  left: 100%;
+
+  height: 100%;
+  width: 100%;
+
+  &__list {
+    list-style: none;
+
+    display: grid;
+    gap: 30px;
+
+    max-width: 1260px;
+    margin: 0 auto;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 22px;
+    color: $color-black;
+    font-weight: 600;
+
+    &--tel {
+      font-size: 16px;
+    }
+
+    &--book {
+      font-size: 12px;
+
+      padding-bottom: 8px;
+      border-bottom: solid 1px $color-black;
+    }
+
+    transition: letter-spacing 0.3s;
+    &:hover {
+      border-bottom: solid 1px $color-black;
```

Comment:
Maybe, let's use the trick with `transparent` border color and change it in hover ?